nexuse2e-webapp/src/main/frontend/src/app/services/session.service.ts

```diff
@@ -57,6 +57,23 @@ export class SessionService {
     }
   }
 
+  getEngineTimeVariables(): EngineTimeVariables | undefined {
+    const engineTimeVariables = sessionStorage.getItem("engine-time-variables");
+    if (engineTimeVariables) {
+      return JSON.parse(engineTimeVariables) as EngineTimeVariables;
+    }
+    return undefined;
```

Comment:
Is it better to return the default value if nothing is found? If we use this on multiple places we can run into having different default values, depending on the page the user visit first. Also we then need no longer a `| undefnied`

Reply:
The value is defined and can be changed in backend, in the frontend we only need this value to display correct and not static labels. We do not want to use a default value (except 0 for not showing this part of the label) because otherwise the label and the result is not the same.